internal/attestation/aws/snp/validator.go

```diff
@@ -31,10 +32,12 @@ import (
 type Validator struct {
 	variant.AWSSEVSNP
 	*vtpm.Validator
-	// amd root key
+	// AMD root key.
 	ark *x509.Certificate
-	// kdsClient is used to query the AMD KDS API.
+	// kdsClient is required for testing.
```

Comment:
```suggestion
	// kdsClient is used to query the AMD KDS API.
```
I would keep the old description, its not relevant for someone reading this code to know that we use this for testing, but its relevant to know what this client does

Reply:
The type's comment has the description you want. It describes what the type does. I think it is useful to tell the reader why that property exists. I can't find where I read that but found that style of comment very helpful while reading new code. I extended the comments, to include the information you are asking for.